src/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ set(HEADERS
 )
 
 add_library(realm_dart SHARED ${SOURCES} ${HEADERS})
-target_compile_definitions(RealmFFIStatic PUBLIC -DRealm_EXPORTS)
+target_compile_definitions(RealmFFIStatic PUBLIC -DRealm_EXPORTS -DREALM_ENABLE_SYNC=1)
```

Comment:
I know I did this, but perhaps @fealebenpae can weigh in, if this is the correct way?

Reply:
No, we shouldn't need to do this. What problem does it solve?

Reply:
OK, I remove -DREALM_ENABLE_SYNC=1 since it is handled in #588